files/puppet_helper.groovy

```diff
@@ -947,11 +947,14 @@ class Actions {
     def jobs = util.findJobs(Jenkins.getInstance())
 
     def allInfo = jobs.collect { path, job ->
+      // at least these job classes do not respond to respond to #isDisabled:
+      // - org.jenkinsci.plugins.workflow.job.WorkflowJob
+      // - com.cloudbees.hudson.plugins.folder.Folder
       def enabled = false
-      // folders don't respond to #isDisabled
-      if (job.getClass().getName() != 'com.cloudbees.hudson.plugins.folder.Folder') {
+      if (job.metaClass.respondsTo(job, 'isDisabled')) {
```

Comment:
I'm considering trying to configure vim to warn about a few rubyisms in groovy code.  If I had a dime for every time I've broken a groovy scipt by using `nil` in stead of `null`...